src/commands/build/download.ts

```diff
@@ -117,16 +118,12 @@ export default class DownloadBuildStatusCommand extends AppCommand {
     return success();
   }
 
-  private downloadFile(uri: string): Promise<ClientResponse<Buffer>> {
-    return new Promise<ClientResponse<Buffer>>((resolve, reject) => {
-      Request.get(uri, { encoding: null }, (error, response, body: Buffer) => {
-        if (error) {
-          reject(error);
-        } else {
-          resolve({ result: body, response });
-        }
-      });
+  private async downloadFile(uri: string): Promise<Response> {
+    const response = await fetchWithOptions(uri, {
```

Comment:
Should we add some error handling here?

Reply:
This method is just a proxy to node-fetch lib. There is an error handling for this download, line 216.

Reply:
It was wrapped with promise before, because Request used callbacks instead of promises. Now it is much simpler.